plugins/youtube_a.js

```diff
@@ -83,7 +83,7 @@ hoverZoomPlugins.push({
             cLog('href=' + href);
 
             const re1 = /\/watch\?v=([^&]{1,})/;   // sample: https://www.youtube.com/watch?v=NaOiA15Rz5k
-            const re2 = /\/youtu.be\/([^&]{1,})/;  // sample: https://youtu.be/qXlQbj0PgDo
+            const re2 = /\/youtu.be\/([^?]{1,})/;  // sample: https://youtu.be/qXlQbj0PgDo https://youtu.be/qORYO0atB6g?t=28
```

Comment:
@GrosPoulet But you are still ignoring the actual time value. If the link points to the middle of the video, your code will start playing it from the beginning. You need to add `#t=...` to video url.

Reply:
Example: https://www.reddit.com/r/videos/comments/vdg6f7/beastie_boys_intergalactic/ - it should start playing from 28s mark, not from the very beginning.